presto-main/src/main/java/com/facebook/presto/sql/gen/InCodeGenerator.java

```diff
@@ -75,12 +98,14 @@ public ByteCodeNode generateExpression(Signature signature, ByteCodeGeneratorCon
                 Object object = constant.getValue();
                 constantValuesBuilder.add(object);
```

Comment:
`constantValues` and `constantValuesBuilder` are not needed when `areSmallPrimitives` are set